vividus-plugin-saucelabs/src/main/java/org/vividus/selenium/sauce/SauceConnectOptions.java

```diff
@@ -98,8 +103,25 @@ public String build(String tunnelIdentifier) throws IOException
 
     private Path createPacFile(String tunnelIdentifier) throws IOException
     {
+        List<String> skipProxyHosts = getSkipProxyHosts();
+        skipProxyHosts.addAll(SAUCELABS_HOSTS);
         return createTempFile("pac-saucelabs-" + tunnelIdentifier, ".js",
-                String.format(PAC_FILE_CONTENT_FORMAT, skipProxyHostsPattern, getProxy()));
+                String.format(PAC_FILE_CONTENT_FORMAT, getSkipShExpMatcher(skipProxyHosts), getProxy()));
+    }
+
+    private String getSkipShExpMatcher(List<String> skipProxyHosts)
+    {
+        return skipProxyHosts.stream()
+                             .map(host -> String.format("shExpMatch(host, \"%s\")", host))
+                             .collect(Collectors.joining(" || "));
```

Comment:
```suggestion
                             .collect(Collectors.joining(" || ", "shExpMatch(host, \"", "\")" ));
```

Reply:
what should I get as an output in this case?

Reply:
please check https://developer.mozilla.org/en-US/docs/Web/HTTP/Proxy_servers_and_tunneling/Proxy_Auto-Configuration_PAC_file#shexpmatch rules for matching